### DISCUSSION TITLE
I configuration my custom provider, but config is not effect

### DISCUSSION THREAD

**Discussion by a137872798** [keycloak/keycloak#21993] [Q&A] [answered]
I need to develop a customer authenticator provider and authenticator  provider factory. When i finish develop work. And put it on META-INF/services.   I start application by docker and i write on standalone.xml. But when application start. I observe logs found config is ineffect. what should i do ?

**Comment by a137872798** [comment]
Here is the part I added
![image](https://github.com/keycloak/keycloak/assets/35627706/20e9e52a-efac-4a59-be10-ef3fbfcc0c31)

**Comment by danielFesenmeyer** [comment, marked as answer]
You should probably update to the latest keycloak version.
Wildfly support (which uses that xml config) has already been removed with keycloak 20, and the current version is 22.
